src/app/shared/modules/header/header.component.spec.ts

```diff
@@ -212,6 +212,29 @@ describe('Header', () => {
     });
   });
 
+  describe('Create Product button', () => {
+    it('should be displayed by default', async () => {
+      const createProductButton = await harness.getCreateProductBtn();
+      expect(createProductButton).not.toBeNull();
```

Comment:
Please use toBeDefined() - it is more straightforward. 

Update here and bellow.

Reply:
Updated in the code and hence submitting the pr.
Merging to master.